usl_pipeline/tests/formats/test_geotiff_loader.py

```diff
@@ -0,0 +1,35 @@
+import unittest
+from usl_pipeline.formats import geotiff_loader
+from rasterio import CRS
+
+
+class GeodiffLoaderTest(unittest.TestCase):
```

Comment:
We can use the pytest style without the need for unnittest subclasses ala
https://docs.pytest.org/en/8.0.x/#a-quick-example